WalletWasabi.Gui/Tabs/WalletManager/WalletManagerViewModel.cs

```diff
@@ -151,12 +151,10 @@ private async Task RefreshHardwareWalletListAsync()
 							else if (hwis.Any(x => x.Type == HardwareWalletType.Ledger && x.Ready))
```

Comment:
Can remove the else too. Easier to read.

Reply:
I agree.